performance/Makefile

```diff
@@ -94,7 +94,7 @@ run-local:
 	docker run \
 		--name perf-tests \
 		$(IMAGE) run \
-		-e DET_MASTER=http://localhost:8080 \
+		-e DET_MASTER=$(DET_MASTER) \
```

Comment:
rm run local now?

Reply:
(i can pr it)

Reply:
i mean get rid of run local since it is identical to run now?

Reply:
ooooooooooh